packages/components/package.json

```diff
@@ -39,6 +39,7 @@
 		"@emotion/css": "^11.11.2",
 		"@emotion/react": "^11.11.1",
 		"@emotion/styled": "^11.11.0",
+		"@storybook/addon-actions": "^8.6.9",
```

Comment:
Why is this part of the PR? Could be nice to include some explanation in the PR description.